interface/new/new_comprehensive_save.php

```diff
@@ -28,11 +28,9 @@
 // other interfaces can still read the data during this lock, however
 // sqlStatement("lock tables patient_data read");
 
-$result = sqlQuery("SELECT MAX(pid)+1 AS pid FROM patient_data");
-
+$result = sqlQuery("SELECT id AS pid FROM patient_data ORDER BY id DESC LIMIT 1");
```

Comment:
@juggernautsei , the only script that should be touched in this PR is interface/new/new_comprehensive_save.php